flatsurf/geometry/gl2r_orbit_closure.py

```diff
@@ -1235,7 +1235,7 @@ def __reduce__(self):
             sage: S = similarity_surfaces.billiard(T)
             sage: S = S.minimal_cover(cover_type="translation")
             sage: O = GL2ROrbitClosure(S)  # optional: pyflatsurf
-            sage: loads(dumps(O)) == O
+            sage: loads(dumps(O)) == O  # long time (6s)
```

Comment:
Here I don't understand what could take so long. There is no subtle algorithm that is run to rebuild `O`.

Reply:
I think that booting the pickle machinery for C++ types is taking a very long time.

Reply:
This is now #123.